src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingleadingasterisk/InputMissingLeadingAsterisk.java

```diff
@@ -0,0 +1,76 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingleadingasterisk;
+
+/** Config = default */
+public class InputMissingLeadingAsterisk {
```

Comment:
please add `// violation` comment to each line where violation is reported

Reply:
Done